dbt_platform_helper/providers/version.py

```diff
@@ -42,38 +53,53 @@ def get_semantic_version(tool_name: str) -> SemanticVersion:
             raise InstalledToolNotFoundException(tool_name)
 
 
-# TODO add timeouts and exception handling for requests
-# TODO Alternatively use the gitpython package?
 class GithubLatestVersionProvider(VersionProvider):
     @staticmethod
-    def get_semantic_version(repo_name: str, tags: bool = False) -> SemanticVersion:
-        if tags:
-            tags_list = requests.get(f"https://api.github.com/repos/{repo_name}/tags").json()
-            versions = [SemanticVersion.from_string(v["name"]) for v in tags_list]
-            versions.sort(reverse=True)
-            return versions[0]
+    def get_semantic_version(
```

Comment:
Does this better describe the intent?
```suggestion
    def get_latest_semantic_version(
```

Reply:
Ignore, I see that's covered in the class name.

Reply:
I think I will leave it as get_semantic_version as this is a common function used between Version providers which also cover installed versions which wouldn't make sense. I think the fact the Provider classes have `Latest` in them should be enough?